### ISSUE TITLE
Required packages not found in Windows

### ISSUE THREAD

**Issue by darkoverlordofdata** [rstudio/shiny#1467]
I'm new to R. I installed RStudio in Linux, and created a new Shiny Web App through the new project menu. Without making any changes, I click on run application. It loads some dependencies and works great. 

I try the same thing in Windows10, and it doesn't work, RStudio fails with 'Required packages versions not found... sourcetools 0.1.5 and html tools 0.3.5... I've tried installing them but it doesn't seem to find them. 

But since it works as expected in Linux, but not Windows (same machine, same versions of RStudio) I have to suspect that the windows install is broken? Or should does Shiny only work in Linux?

**Comment by wch** [comment]
That's strange. It sounds as your R on Windows may be configured to use a different CRAN (package) repository than normal. Have you taken a look at this?
https://support.rstudio.com/hc/en-us/articles/200554786-Problem-Installing-Packages

The CRAN mirror you should be using should be listed as "Global (CDN) - RStudio".

What version of R and RStudio are you using?

**Comment by darkoverlordofdata** [comment]
I found the problem I installed this on 2 other Windows PCs. One fails the same, one works, like Linux. The difference? VisualStudio was not installed on the machine where it works. It also asked me to let it install libraries. The machines that have VS don't ask to install libs, they just fail.  This is the latest versions, studio 1.0.44 and R version 3.2.2. It's definitly environment, I just have to figure out how to get VS to stop interfering with RStudio. Typical for M$.
